README.md

```diff
@@ -1,6 +1,14 @@
 # AI Multimodal Public Demos
 
-The demos in this repostiory rely solely on public data sets.
+This repository is part of [AIMM project](https://github.com/AI-multimodal): a joint and collaborative effort between scientist from [Brookhaven National Laboratory (BNL)](https://www.bnl.gov/) and [Lawrence Berkeley National Laboratory (LBL)](https://www.lbl.gov/)
```

Comment:
We should include all institutions involved in the grant.

Reply:
Should we include facilities as well as labs?

Labs: BNL, ANL, LBNL
Facilities: NSLS-II, APS, ALS, CNM, CFN

Right?

Reply:
@dylanmcreynolds  I can group the facilities by Lab. It would make it easier for the general public to understand how we are organized. Are these all of the facilities involved in the project?